macOS/DuckDuckGo/TabBar/ViewModel/TabCollectionViewModel.swift

```diff
@@ -124,6 +124,10 @@ final class TabCollectionViewModel: NSObject {
         return homePage
     }
 
+    /// This property logic will be true when the user appends a new tab
+    /// it will be set to false when the user moves selects an existing tab
```

Comment:
Just making sure the docs are clear

```suggestion
    /// it will be set to false when the user selects an existing tab
```